lib/client.js

```diff
@@ -62,7 +62,10 @@ class Client {
 
   parse_options (options) {
     const api_key = options.api_key || process.env['__OW_API_KEY']
-    const ignore_certs = options.ignore_certs || false
+    const ignore_certs = options.ignore_certs ||
+      (process.env['__OW_IGNORE_CERTS']
```

Comment:
You cloud write it shorter 
`(process.env['__OW_IGNORE_CERTS'] && process.env['__OW_IGNORE_CERTS'].toLowerCase() === "true") ` not need the ternary operator

Reply:
That ended up defaulting to `undefined`